src/Validators/IntegrityChecks/Form/ConditionalElementCheck.cs

```diff
@@ -23,7 +23,7 @@ public IntegrityCheckResult Validate(FormSchema schema)
 
             List<IElement> conditionalElements = schema.Pages
                 .Where(page => page.Elements is not null)
-                .SelectMany(page => page.Elements)
+                .SelectMany(page => page.ValidatableElements)
```

Comment:
this needs to be changed back to just Elements, as u can show non validatable element in conditional e.g. p or alert